### ISSUE TITLE
Group Convolution

### ISSUE THREAD

**Issue by pGit1** [scheckmedia/keras-shufflenet#3]
@scheckmedia ,

How fast is your grouped Convolution code? I've seen the lambda implementation elsewhere and it was pretty slow. 

Do you have similar results?

Thanks for this great implementation

**Comment by scheckmedia** [comment]
Hi,

I think the grouped convolution is still a bottleneck. The training took more time than the training with the MobileNet.

If I find some free minutes I will measure the inference time for the MobileNet and the ShuffleNet. I will update the Readme if I have the values.

**Comment by scheckmedia** [comment]
The following URL  contains the Tensorflow benchmark values for the MobileNet 0.75 compared to the ShuffleNet 1.5X. 

https://gist.github.com/scheckmedia/6e93dd1eb45ef47eb870dccfe473d319

It seems that the ShuffleNet has a better inference time than the MobileNet.

**Comment by pGit1** [comment]
Thanks @scheckmedia !
